src/pages/introduction/essentials/tech-stack/index.md

```diff
@@ -0,0 +1,47 @@
+# Programming languages
```

Comment:
@anoopvalomkot  What are the system requirements for InDesign to work smoothly? Specific OS versions etc.

Reply:
@padmkris123 
https://helpx.adobe.com/in/indesign/system-requirements.html
https://helpx.adobe.com/indesign/kb/system-requirements-indesign-server.html